core/lib/dr_tools.h

```diff
@@ -48,6 +48,16 @@ DR_API
  * \warning This context cannot be used as the drcontext for a thread
  * running under DR control!  It is only for standalone programs that
  * wish to use DR as a library of disassembly, etc. routines.
+ * \warning This context is not fully thread-safe as it stores some state
+ * (such as dr_isa_mode_t and other fields related to AArch32 encoding
+ * and decoding) that is global and may be prone to data races.
+ * For example, having different threads use dr_set_isa_mode() to set
+ * different ISA modes at the same time can result in a data race.
+ * Furthermore, encoding and decoding of AArch32 instructions in parallel
+ * may also result in a data race.
+ * Code that uses stanalone DR context across multiple threads should
+ * implement its own lock/unlock mechanism to avoid such data races
+ * when using dr_set_isa_mode() or encoding/decoding AArch32 instructions.
```

Comment:
Outside of the Doxygen comment (so it won't appear in the html docs) add a (regular) comment `TODO i#6690: Add better multi-thread standalone decoding support` or sthg like that.

Reply:
Done.

Reply:
You can check if it renders as expected in the generated html file.

Reply:
Done. It's not in the html.